maestrowf/specification/yamlspecification.py

```diff
@@ -158,16 +151,25 @@ def load_specification_from_stream(cls, stream):
 
     def verify(self):
         """Verify the whole specification."""
-        self.verify_description()
-        self.verify_environment()
-        self.verify_study()
-        self.verify_parameters()
+
+        # load the YAMLSpecification schema file
+        classname = self.__class__.__name__.lower().replace(".py", "")
+        dirpath = os.path.dirname(os.path.abspath(__file__))
+        schema_path = os.path.join(dirpath, "schemas")
+        schema_path = os.path.join(schema_path, "{}.json".format(classname))
```

Comment:
Hardcoding this to `schema_path = os.path.join(schema_path, "yamlspecification.json")` worked -- I'm starting to think this is okay, since the verification is scoped to each specification. The `schema` folder you introduced means that there's a central place for the json files now.

Reply:
Sounds good.